src/for-students-alumni/Alumni.tsx

```diff
@@ -0,0 +1,44 @@
+import { Center, Title, Grid, Image } from "@mantine/core";
+import "./Alumni.css";
+
+const companies = [
+  { src: "/logos/meta.svg", alt: "Meta" },
+  { src: "/logos/google.svg", alt: "Google" },
+  { src: "/logos/stripe.svg", alt: "Stripe" },
+  { src: "/logos/microsoft.svg", alt: "Microsoft" },
+  { src: "/logos/amazon.svg", alt: "Amazon" },
+  { src: "/logos/intuit.svg", alt: "Intuit" },
+  { src: "/logos/intel.svg", alt: "Intel" },
+  { src: "/logos/figma.svg", alt: "Figma" },
+  { src: "/logos/coursera.svg", alt: "Coursera" },
+  { src: "/logos/rbc.svg", alt: "RBC" },
+  { src: "/logos/citadel.svg", alt: "Citadel" },
+  { src: "/logos/square.svg", alt: "Square" },
+];
+
+function Alumni() {
+  return (
+    <div className="alumni">
+      <Center>
+        <Title className="alumni-title" size="3vw">
+          Our Alumni have gone on to innovate at:
+        </Title>
```

Comment:
This starts to overflow when in mobile view. Can probably make it two lines when the viewport gets small enough

Reply:
having trouble replicating this. could you please share a screenshot?